### ISSUE TITLE
Document the input format of MultinomialHMM

### ISSUE THREAD

**Issue by superbobry** [hmmlearn/hmmlearn#125]
See #124 for the problem.

I think adding a proper example to "Examples" section might be enough.

**Comment by owen-jones-gen** [comment]
I agree with this. I am trying to get it working at the moment. Issue #70 is useful, though you have to read it all the way through to find bugs that were spotted at the end.
